### ISSUE TITLE
arguments with dashes cant be variables in bash

### ISSUE THREAD

**Issue by nikita-skobov** [nikita-skobov/create-bash-script#5]
a lot of command line programs and scripts use arguments with dashes:
`--ignore-warnings` or `--exec-path`, etc.

Our create-bash-script will literally take these arguments and treat the whole string as a variable, creating a variable named: `exec-path` for example. Bash cannot have dashes in variable names, so I propose a change to rename variables with dashes to use underscores instead. ie: instead of generating argument parsing like this:
```sh
case $key in
    --test-arg)
        test-arg="$2"
        shift 2
        ;;
```

it should generate argument parsing like this:
```sh
case $key in
    --test-arg)
        test_arg="$2"
        shift 2
        ;;
```

**Comment by bronweg** [comment]
Wow!
You are right, I haven't thought about it.
I will try to manage it over the weekend.

**Comment by nikita-skobov** [comment]
No worries, I think I fixed it in #6 

https://github.com/nikita-skobov/create-bash-script/pull/6

Please review it to make sure I didn't break any of your code. I ran a few tests and it seems to work fine, but I'd like to get your confirmation before merging to master. If everything looks good just leave a comment on the PR and I'll merge it and then close this issue.
